generate_engagement_analysis_files.py

```diff
@@ -103,3 +103,60 @@
 
         for row in opted_in_uids_per_show.items():
             writer.writerow(row)
+
+    log.info(f'Computing repeat and new participation per show ...')
+    # Computes the number of new and repeat consented individuals who participated in each radio show.
+    # Repeat participants are consented individuals who participated in previous shows prior to the target show.
+    # New participants are consented individuals who participated in target show but din't participate in previous shows.
+    repeat_new_participation_map = OrderedDict() # of rqa_raw_field to participation metrics.
+    for rqa_raw_field in rqa_raw_fields:
+        target_radio_show = rqa_raw_field # radio show in which we are calculating participation metrics for.
+
+        target_radio_show_uids = [] # contains uids of individuals who participated in target radio show.
+        with open(f'{demog_map_json_input_dir}/{target_radio_show}_demog_map.json') as f:
+            target_radio_show_data = json.load(f)
+            for uid in [*target_radio_show_data]:
```

Comment:
What does the '*' do? Is it a list spread operator for Python?

If so please flag this to Luke on his review as this a new language feature which we haven't used in our code base yet

Reply:
If this is a spread operator I'm not sure it's needed here, you should just be able to `for uid in target_radio_show_data`

Reply:
It unpacks the sequence but I have actually realized we don't need it. Thanks for pointing it out.